backend/src/federation/index.ts

```diff
@@ -4,11 +4,22 @@
 import DHT from '@hyperswarm/dht'
 // import { Connection } from '@dbTools/typeorm'
 import { backendLogger as logger } from '@/server/logger'
+import CONFIG from '@/config'
 
 function between(min: number, max: number) {
   return Math.floor(Math.random() * (max - min + 1) + min)
 }
 
+const getSeed = (): Buffer | null => {
+  if (CONFIG.FEDERATION_DHT_SEED) {
+    logger.debug(`with seed='${CONFIG.FEDERATION_DHT_SEED}'`)
+    return Buffer.alloc(KEY_SECRET_SEEDBYTES, CONFIG.FEDERATION_DHT_SEED)
+  }
+  return null
+}
```

Comment:
excluding the log this can be reduced to a one-liner. I agree to have this in a function with the null value as option.

```suggestion
const getSeed = (): Buffer | null => CONFIG.FEDERATION_DHT_SEED ? Buffer.alloc(KEY_SECRET_SEEDBYTES, CONFIG.FEDERATION_DHT_SEED) : null
```

Reply:
good solution, i've tried some expressions with such a one-line logic, but they doesn't work as expected...
now used your one!